cfn-resources/cluster/cmd/resource/resource.go

```diff
@@ -389,9 +389,6 @@ func clusterCallback(client *mongodbatlas.Client, currentModel *Model, projectID
 }
 
 func (m *Model) HasAdvanceSettings() bool {
-	/*TODO: this logic is because of a bug un Cloud Formation, when we return in_progress in the CREATE
-	,the second time the CREATE gets executed
-	it returns the AdvancedSettings is not nil but its fields are nil*/
```

Comment:
I think we should keep the message to explain why there is this logic. Just remove the `TODO` from it

Reply:
agreed! ill just add a ticket so we can keep track of this issue, ill try to create one on AWS if i dont het an answer on chime